WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -56,13 +56,15 @@ public final class SingleProductScreen: ScreenObject {
             XCTAssertTrue(app.staticTexts["Add variations"].exists)
         case "grouped":
             XCTAssertTrue(app.staticTexts["Grouped products"].exists)
+        case "external":
+            XCTAssertTrue(app.staticTexts["Add product link"].exists)
         default:
             XCTFail("Product Type \(productType) doesn't exist!")
         }
     }
 
     public func verifyProductTypeScreenLoaded(productType: String) throws -> Self {
-        let productTypeLabel = NSPredicate(format: "label ==[c] '\(productType)'")
+        let productTypeLabel = NSPredicate(format: "label ==[c] '\(productType)\(productType == "external" ? "/Affiliate" : "")'")
```

Comment:
Even though it's understandable on an intuitive level, this expression is rather intricate once you need to really parse it, because of the amount of `\`, `/`, `'` and `"`. I can propose this instead:

```suggestion
        let productTypeLabel = productType + (productType == "external" ? "/Affiliate" : "")
        let productTypeLabelPredicate = NSPredicate(format: "label ==[c] '\(productTypeLabel)'")
```

(this is working code, but note that I renamed the original `productTypeLabel` to `productTypeLabelPredicate`, which should also be updated on L71)

Reply:
Thanks for the suggestion! Yeah I'll use your suggestion to break it down a bit more. Only reason I'm not using your commit is to also update the other line to use the new `productTypeLabelPredicate` value.